appnet/appnet.php

```diff
@@ -296,7 +296,7 @@ function appnet_post_local(&$a,&$b) {
 		$appnet_enable = (($appnet_post && x($_REQUEST,'appnet_enable')) ? intval($_REQUEST['appnet_enable']) : 0);
 
 		// if API is used, default to the chosen settings
-		if($_REQUEST['api_source'] && intval(get_pconfig(local_user(),'appnet','post_by_default')))
+		if($b['api_source'] && intval(get_pconfig(local_user(),'appnet','post_by_default')))
```

Comment:
shoudn't there be curly brackets?

Reply:
Since app.net doesn't work anymore, the change there doesn't matter.

Reply:
Then see the blogger changes ;-)

Reply:
I guess that blogger doesn't work as well, since for a long time they now belong to Google and authentication was moved to OAuth.